<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomizeDialog</class>
 <widget class="QDialog" name="CustomizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>362</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Browser customization</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Color selection</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="ColorButton" name="background_cbutton" native="true"/>
          </item>
          <item>
           <widget class="ColorButton" name="bborder_cbutton" native="true"/>
          </item>
          <item>
           <widget class="ColorButton" name="line_cbutton" native="true"/>
          </item>
          <item>
           <widget class="ColorButton" name="text_cbutton" native="true"/>
          </item>
          <item>
           <widget class="ColorButton" name="bfill_cbutton" native="true"/>
          </item>
          <item>
           <widget class="ColorButton" name="hborder_cbutton" native="true"/>
          </item>
          <item>
           <widget class="ColorButton" name="hfill_cbutton" native="true"/>
          </item>
          <item>
           <widget class="ColorButton" name="selection_cbutton" native="true"/>
          </item>
          <item>
           <widget class="ColorButton" name="link_cbutton" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="groupBox1_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Icon size</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QSlider" name="iconSize_slider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="iconSize_spin">
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="toolTip">
             <string>Set the icon size</string>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>123</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" alignment="Qt::AlignLeft">
        <widget class="QGroupBox" name="groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Tree display</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="3" column="0">
           <widget class="QCheckBox" name="centerParent_checkbox">
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="toolTip">
             <string>Should the tree be rendered to minimize space?</string>
            </property>
            <property name="text">
             <string>Center parent</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="compacting_checkbox">
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="toolTip">
             <string>Should the tree be rendered to minimize space?</string>
            </property>
            <property name="text">
             <string>Compacting</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox5">
         <property name="title">
          <string>Font</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="font_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>&lt;p align=&quot;center&quot;&gt;abcdefg&lt;br&gt;
ABCDEFG&lt;/p&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <spacer name="spacer2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="font_button">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="text">
               <string>Font</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer2_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QWidget</extends>
   <header>ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">ColorButton.h</include>
  <include location="local">ColorButton.h</include>
  <include location="local">ColorButton.h</include>
  <include location="local">ColorButton.h</include>
  <include location="local">ColorButton.h</include>
  <include location="local">ColorButton.h</include>
  <include location="local">ColorButton.h</include>
  <include location="local">ColorButton.h</include>
  <include location="local">ColorButton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
